internal/protoschema/jsonschema/jsonschema.go

```diff
@@ -231,15 +231,15 @@ func (p *jsonSchemaGenerator) generateEnumValidation(field protoreflect.FieldDes
 
 func (p *jsonSchemaGenerator) generateIntValidation(_ protoreflect.FieldDescriptor, entry map[string]interface{}, bitSize int) {
 	// Use floats to handle integer overflow.
-	min := -math.Pow(2, float64(bitSize-1))
-	max := math.Pow(2, float64(bitSize-1))
+	minSize := -math.Pow(2, float64(bitSize-1))
+	maxSize := math.Pow(2, float64(bitSize-1))
```

Comment:
This failed [predeclared](https://golangci-lint.run/usage/linters/#predeclared) linting in Go v1.23.